src/controller/systeminfo/controller.go

```diff
@@ -50,6 +51,7 @@ type Data struct {
 }
 
 type protectedData struct {
+	CurrentTime                 time.Time
```

Comment:
why it is an protected data?

Reply:
because the current_time is only added to the response of systeminfo API when user is logged in.